extensions/kusto/config.json

```diff
@@ -1,13 +1,15 @@
 {
-	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/v{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
-	"version": "3.0.0-release.133",
+	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/{#version#}/microsoft.sqltools.servicelayer-{#fileName#}",
+	"version": "4.11.0.2",
```

Comment:
Updating this seems necessary to resolve hygiene errors - will continue testing on multiple platforms as we stabilize this extension further. No concerns on Windows so far.

Reply:
Validated on macOS ARM as well.